ci/build-msys2.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh -e
+
+if [ "$1" = "meson" ]; then
+    meson setup build_meson      \
+      -D cdda=enabled            \
+      -D d3d-hwaccel=enabled     \
+      -D d3d11=enabled           \
+      -D dvdnav=enabled          \
+      -D egl-angle-win32=enabled \
+      -D jpeg=enabled            \
+      -D lcms2=enabled           \
+      -D libarchive=enabled      \
+      -D libbluray=enabled       \
+      -D libmpv=true             \
+      -D libplacebo=enabled      \
+      -D lua=luajit              \
+      -D pdf-build=enabled       \
+      -D rubberband=enabled      \
+      -D shaderc=enabled         \
+      -D spirv-cross=enabled     \
+      -D tests=true              \
+      -D uchardet=enabled        \
+      -D vapoursynth=enabled     \
+      -D vulkan=enabled
```

Comment:
does msys have bash?
you could shorten this a lot using `-D{cdda,d3d-hwaccel,...}=enabled` syntax

Reply:
I think it having it organized in multi line is more readable, than putting everything in one line expansion. Can't really break lines, because then indentation can't be done and everything looks ugly.